lib/views/pages/splash_screen.dart

```diff
@@ -0,0 +1,98 @@
+import 'dart:async';
```

Comment:
suggesting to move the file inside session folder since they're somewhat related